parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -83,6 +83,8 @@ if [[ -n "${OUTBOUND_COMMAND}" ]]; then
         eval $PROXY_VARS
     fi
     retrycmd_if_failure 50 1 5 $OUTBOUND_COMMAND >> /var/log/azure/cluster-provision-cse-output.log 2>&1 || exit $ERR_OUTBOUND_CONN_FAIL;
+else
+    touch /var/run/outbound-check-skipped
```

Comment:
add some comment to indicate what's this file for?
e.g.

`# This fie indicates the cluster doesn't have outbound connectivity and should be excluded in future external outbound checks`

Reply:
added